source/tyr/tyr/resources.py

```diff
@@ -1634,3 +1636,43 @@ def put(self, instance_name):
             return_status = 404
 
         return {'action': return_msg}, return_status
+
+
+class Cities(flask_restful.Resource):
+    def check_db(self):
+        cities_db = sqlalchemy.create_engine(current_app.config['CITIES_DATABASE_URI'])
```

Comment:
You could start by checking that `CITIES_DATABASE_URI` isn't `None`

Reply:
:heavy_check_mark: